lib/y2lan_restart_common.pm

```diff
@@ -150,12 +150,14 @@ sub check_network_status {
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Do not expect Network restart/reload. journalctl cannot find events");
     }
     else {
-        assert_equals($expected_status, 'restart', "Network restart occurred, even though it wasn't expected.");
+        my $journal_findings = script_output('cat journal.log');
+        my $actual_status    = is_sle('>=15-SP2') ? 'reload' : 'restart';
```

Comment:
in current implementation this switch does not make any sense. In a fact we don't care if it was restart or reload . Here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9945/files#diff-b0dd6307080be5c53e07144fed58bbf4R74 we clearly saying if **something** was grep'ed from journal say yes . So in current implementation we just checking that we pickup correct $expected_status in **calling test** ( see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9945/files#diff-ef1d03708a8de90e3bef67901eb2f650R89)  but not in **system under test**. So I see two options : 
1. Drop this distinction here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9945/files#diff-ef1d03708a8de90e3bef67901eb2f650R89 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9945/files#diff-ef1d03708a8de90e3bef67901eb2f650R89 . And just have some alias which would mean "restart or reload". No matter what actual name it will have 
2. Introduce same distinction here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9945/files#diff-b0dd6307080be5c53e07144fed58bbf4R74 so we will really fail when in SLE 15 SP1 reload will happen

Reply:
Also, please note that TW and leap have same behavior as SLE 15 SP2, so it means that it's better to use opposite condition like `is_sle('<=15-SP1')`. But it really doesn't make sense if we don't adjust `initialize_y2lan` and/or `check_network_status` methods.

Reply:
Instead of created an alias i approach it a slightly different from the suggestions. There is still a distrinction [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9945/files#diff-ef1d03708a8de90e3bef67901eb2f650R98) and then i added another assert to cover and distinguish the reload. So if the journal.log contains one of the shutting down|ifdown all|Stopping wicked this means that the expected action should match with restart. in case that Reloaded wicked is in the file, it means that the service has perform a reload and assert for it.